### PR TITLE
feat(data table manager): add property to disable dropdown options in…

### PR DESCRIPTION
**Pull request by ddouglasz**
## Description 
During the migration to the dropdown options for the data table manager, we missed the proper behaviuor of the data table when disabled

This pull request properly introduces the expected feature to the data table manager component, allowing dropdown options to be disabled. .

### CODE REVIEW

File: packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ type MappedColumns = Record<string, TColumnData>;
 export type TDropdownOption = {
   value: string;
   label: string;
+  isDisabled?: boolean;
```
Comment (misama-ct) [inline packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx:37]: Not sure if this shows up in the README.md, but maybe just run `yarn generate-readmes` to figure out if something changed and commit if necessary.

**Review by misama-ct — APPROVED** [review]
Thank you!

File: packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx [hunk 2]
```diff
@@ -88,7 +101,15 @@ export const getDropdownOptions = ({
               : formatMessage(messages.displaySettingsOption),
           },
         ]
-      : []),
+      : [
+          {
+            value: DISPLAY_SETTINGS,
+            isDisabled: true,
```
[the remark this answers is not in the document]
Comment (ddouglasz) [reply packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx:107]: Good question. In a sense, we are not hard coding it. When you look at the logic that leads to it, we already did a check in a function where the disabled property is passed from the customSettings and then further used to check if we should disable or not  
```
const areCustomColumnSettingsEnabled = Boolean(
    props.customColumnManager &&
      !props.customColumnManager?.disableCustomColumnManager
  );
```

[the remark this answers is not in the document]
Comment (obulaworld) [reply packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx:107]: @ddouglasz Okay I understand now. So we will now have to make use of the `areCustomColumnSettingsEnables` prop.

**Review by obulaworld — APPROVED** [review]

File: packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ddouglasz) [reply packages/components/data-table-manager/src/data-table-settings/data-table-settings.tsx:107]: We will pass the `disableCustomColumnManager` through the `customColumnManager` object.

**Pull request merged**